sustainability_hr_expense_report/views/hr_expense.xml

```diff
@@ -10,22 +10,9 @@
                 <field name="carbon_is_locked" invisible="1" />
                 <label for="carbon_debt" />
                 <div class="o_row">
-                    <field name="carbon_debt" digits="[13,2]" attrs="{'readonly': [('carbon_is_locked', '=', True)]}" />
-                    <button
-                        title="Unlock co2"
-                        type="object"
-                        name="action_switch_locked"
-                        attrs="{'invisible': [('carbon_is_locked', '=', True)]}"
-                        icon="fa-unlock"
-                        class="opacity-25"
-                    />
-                    <button
-                        title="Lock co2"
-                        type="object"
-                        name="action_switch_locked"
-                        attrs="{'invisible': [('carbon_is_locked', '=', False)]}"
-                        icon="fa-lock"
-                    />
+                    <field name="carbon_debt" digits="[13,2]" readonly="carbon_is_locked" />
+                    <button title="Unlock co2" type="object" name="action_switch_locked" invisible="carbon_is_locked" icon="fa-unlock" class="opacity-25" />
+                    <button title="Lock co2" type="object" name="action_switch_locked" invisible="not carbon_is_locked" icon="fa-lock" />
```

Comment:
to improve the ux it's better if the condition is inversed, like we did for sustainability module: https://github.com/sustainability-suite/sustainability-odoo/commit/bb3a8bb603ff2b8d978d7fed29909dd60f6a122d#diff-9d77a3a4b0a4690af9ab442a283d500c2500292eb606c04ec412387d1050108bR20

```suggestion
                    <button title="Unlock co2" type="object" name="action_switch_locked" invisible="not carbon_is_locked" icon="fa-unlock" class="opacity-25" />
                    <button title="Lock co2" type="object" name="action_switch_locked" invisible="carbon_is_locked" icon="fa-lock" />
```

Reply:
yes thanks